simtools/utils/general.py

```diff
@@ -369,7 +369,7 @@ def collect_data_from_http(url):
     return data
 
 
-def collect_data_from_yaml_or_dict(in_yaml, in_dict, allow_empty=False):
+def collect_data_from_file_or_dict(in_yaml, in_dict, allow_empty=False):
```

Comment:
I suggest replacing the argument `in_yaml` by `file` or `file_name` as the file is not necessarily a yaml file anymore.

Reply:
Good point - changed it to `file_name`.